src/main/java/mate/academy/rickandmorty/service/impl/CharacterServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.rickandmorty.service.impl;
+
+import jakarta.annotation.PostConstruct;
+import java.util.List;
+import java.util.Random;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDto;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.mapper.CharacterMapper;
+import mate.academy.rickandmorty.model.Character;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import mate.academy.rickandmorty.service.CharacterClient;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CharacterServiceImpl implements CharacterService {
+    private static Long maxId;
+    private final CharacterClient characterClient;
+    private final CharacterRepository characterRepository;
+    private final CharacterMapper characterMapper;
+    private final Random random = new Random();
```

Comment:
You have spring IoC to avoid instantiation like this
```suggestion
    private final Random random = new Random();
```